build-system/tasks/performance/index.js

```diff
@@ -32,13 +33,17 @@ async function performance() {
   await compileScripts(urls);
   await rewriteScriptTags(urls);
   await getMetrics(urls, {headless, runs});
+  runTests();
   printReport(urls);
 }
 
 performance.description = '  Runs web performance test on current branch';
 
 performance.flags = {
   'nobuild': 'Does not compile javascripts before running tests',
+  'threshold':
+    'Percentage above stable metrics that is passable. Number between 0 and 1',
```

Comment:
For clarity, can you specify if the number is a fraction, or if it's a number in percentage points?

Also, curious: why do we limit it to the 0-1 range?

Reply:
I thought limiting it to 200% was reasonable. I added the limit to clarify that the number is a ..fraction.. ? As in, `threshold=0.5` means the branch can be 150% above stable. Any suggestions to clarify this?

Reply:
Gotcha. How about something like this, to make it clear that it's a floating point number?

```
Fraction by which metrics are allowed to increase. Number between 0.0 and 1.0.
```

Reply:
Thanks! Done